addons/resource.language.en_gb/resources/strings.po

```diff
@@ -38,7 +45,7 @@ msgstr ""
 
 #: Used in Confluence
 msgctxt "#4"
-msgid "TV-Guide"
+msgid "TV-guide"
```

Comment:
I'm almost certain this string is not used anywhere anymore, but either way it should be `TV guide`, not `TV-guide`.